docs/guide/configuration.md

```diff
@@ -129,19 +193,45 @@ Check out [@ts-safeql-demos/multi-connections](https://github.com/ts-safeql/safe
 }
 ```
 
-### What is "connectionUrl" and should I supply it?
+## Example 5: Mixing `databaseUrl` and `migrationsDir` configurations
 
-::: info TL;DR
-If your'e using migrations and your postgres database URL is different than
-```
-postgres://postgres:postgres@localhost:5432/postgres
-```
-Then you have to supply an appropriate `connectionUrl`.
+::: tip DEMO
+Check out [@ts-safeql-demos/multi-connections](https://github.com/ts-safeql/safeql/tree/main/demos/multi-connections) for a working example.
```

Comment:
We don't really have a demo for that. This demo should be in the previous example. I'm okay with not writing an example for specific case.

Reply:
c95ee5282480811e014f1df172f350c8b1551c35

Reply:
Sorry about that! Totally overlooked these demos!